Makefile.in

```diff
@@ -267,6 +267,24 @@ runtest:
 	  ARCH=`grep '^ARCH=' ocaml/Makefile.config | cut -d'=' -f2` \
 	  $(dune) runtest --root=. --profile=release --build-dir=_build2
 
+# Only needed for running the test tools by hand; runtest will take care of
+# building them using Dune
+#
+# CR lmaurer: stage2 should be a dependency here, but I'm wary because
+# there may be a reason runtest doesn't depend on stage2.
```

Comment:
I'm unsure what dependency from stage2 is needed here -- sure, the directory needs to exist, but that doesn't need to be declared here (as far as I can see).